### ISSUE TITLE
Support survival analysis tasks

### ISSUE THREAD

**Issue by RaphaelS1** [christophM/iml#172]
Hi Christoph,

Thanks for making models more interpretable!
Would you be happy to start supporting survival analysis in iml (via mlr3proba)? I'm happy to make a PR for this if you are.

Thanks!

**Comment by christophM** [comment]
What would you use as a prediction in the case of survival analysis?
Do you have a quick proof of concept using a custom predict.fun for Predictor$new()?

**Comment by RaphaelS1** [comment]
> What would you use as a prediction in the case of survival analysis?

Agreed it isn't straight forward and it is messy but in general I'd say the only possible compatible prediction is a continuous relative risk ranking which you would then compute concordance on. Has little meaning in interpreting things like SHAPs (what does a relative increase of a relative prediction even mean) but is useful for feature importance.

> Do you have a quick proof of concept using a custom predict.fun for Predictor$new()?

Can't Predictor just wrap mlr models?

**Comment by Jack-FG** [comment]
It would be nice to create ALE plots in addition to the pdp and marginal effects plots cabaple of being produced with these models: https://www.randomforestsrc.org/articles/partial.html

Random forest survival models
